src/main/java/org/jitsi/videobridge/xmpp/MediaStreamTrackFactory.java

```diff
@@ -174,8 +178,14 @@ else if (temporalIdx > 0)
                     rtpEncodings[idx]
                         = new RTPEncodingDesc(track, idx,
                         primary[streamIdx],
-                        rtx == null || rtx.length == 0 ? -1 : rtx[streamIdx],
                         temporalId, spatialId, height, frameRate, dependencies);
+                    SecondarySsrcs ssrcSecondarySsrcs = secondarySsrcs.get(primary[streamIdx]);
+                    if (ssrcSecondarySsrcs != null)
+                    {
+                        ssrcSecondarySsrcs.forEach(ssrcSecondarySsrc -> {
+                            rtpEncodings[idx].addSecondarySsrc(ssrcSecondarySsrc.ssrc, ssrcSecondarySsrc.type);
```

Comment:
same as above. If we don't check for null, I would love to at least have a comment here.

Reply:
what sort of comment were you thinking?